cmd/api/src/database/oidc.go

```diff
@@ -24,7 +24,17 @@ import (
 
 // OIDCProviderData defines the interface required to interact with the oidc_providers table
 type OIDCProviderData interface {
+	GetOIDCProvider(ctx context.Context, id int64) (model.OIDCProvider, error)
 	CreateOIDCProvider(ctx context.Context, name, issuer, clientID string) (model.OIDCProvider, error)
+	DeleteOIDCProvider(ctx context.Context, id int64) error
+}
+
+func (s *BloodhoundDB) GetOIDCProvider(ctx context.Context, id int64) (model.OIDCProvider, error) {
```

Comment:
I believe we'll also need a get method for fetching via name / slug as well for the login flow.

Reply:
Disregard, added in a the login PR